.github/workflows/host-image-ci.yml

```diff
@@ -0,0 +1,337 @@
+    - name: Inject name and ID into preseed 
+      run: echo "CI_PROJECT_NAME=${{ github.event.repository.name }}" >> tools/cuttlefish-host-image-installer/preseed/after_install_1.sh && echo "CI_PIPELINE_ID=${{ github.repository}}/${{ github.workflow }}" >> tools/cuttlefish-host-image-installer/preseed//after_install_1.sh
+    - name: Download Debian installer
+      run: cd tools/cuttlefish-host-image-installer && wget -nv -c ${DEBIAN_ISO_URL} && ./addpreseed.sh && xz -9e preseed-mini.iso
+    - name: Publish preseed-mini.iso.xz
+      uses: actions/upload-artifact@v3
+      with:
+        name: preseed-mini
+        path: tools/cuttlefish-host-image-installer/preseed-mini.iso.xz
+
+  build-cuttlefish-deb-job:
+    uses: ./.github/workflows/presubmit.yaml
+
+  build-metapackage-deb-job:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout repository
+      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
+    - name: Prepare pbuilder environment
+      run: sudo apt update && sudo apt-get install -y ubuntu-dev-tools qemu-user-static binfmt-support fakeroot equivs      
+    - name: Prepare building environment
+      run: sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get install -y debconf-utils
+    - name: Prepare tzdata
+      run: echo "tzdata tzdata/Areas select Etc" | sudo debconf-set-selections -v && echo "tzdata tzdata/Zones/Etc select UTC" | sudo debconf-set-selections -v && DEBIAN_FRONTEND=noninteractive sudo apt-get install -y tzdata && sudo dpkg-reconfigure --frontend noninteractive tzdata
+    - name: Prepare pbuilder tool
+      run: echo "pbuilder pbuilder/mirrorsite string http://deb.debian.org/debian/" | sudo debconf-set-selections -v && DEBIAN_FRONTEND=noninteractive sudo apt-get install -y pbuilder && sudo dpkg-reconfigure --frontend noninteractive pbuilder
+    - name: Prepare other tools
+      run: sudo apt-get install -y ubuntu-dev-tools qemu-user-static && sudo apt-get install -y git ca-certificates less && sudo apt-get install -y debhelper devscripts cdbs dpkg-dev equivs fakeroot && sudo apt-get install -y build-essential autoconf automake && sudo apt-get install -y p7zip-full
+    - name: Pbuilder arm64 create
+      run: pbuilder-dist stable arm64 create
+    - name: Pbuilder arm64 update
+      run: pbuilder-dist stable arm64 update
+    - name: Building package
+      run: cd tools/cuttlefish-host-image-installer/metapackage-google && pdebuild --buildresult .. -- --basetgz ~/pbuilder/stable-arm64-base.tgz
+    - name: Create artifact
+      run: cd tools/cuttlefish-host-image-installer && 7z a -mx=9 meta_gigamp_packages.7z *.deb
+    - name: Publish meta_gigamp_packages.7z
+      uses: actions/upload-artifact@v3
+      with:
+        name: meta_gigamp_packages
+        path: tools/cuttlefish-host-image-installer/meta_gigamp_packages.7z
+
+  build-kernel-aosp14-61-deb-job:
```

Comment:
`build-kernel-aosp14-61-deb-job` and `build-kernel-aosp15-61-deb-job` and `build-kernel-aosp15-66-deb-job` look the same to me except for different `KERNEL_MANIFEST_BRANCH` values, can they use a [matrix strategy](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) to vary the `KERNEL_MANIFEST_BRANCH` value without copy-pasting the `steps`?

The jobs that vary with the `matrix` might need to be in their own file so that the matrix doesn't duplicate the jobs that don't vary.

Reply:
I don't know matrix strategy at that time. Let me figure out how to use it. Thanks!

Reply:
Done.